.github/workflows/codeformat.yaml

```diff
@@ -27,20 +27,14 @@ jobs:
           echo "After clearing disk space:"
           df -h
       - name: Set up Go 1.18.1
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
         with:
           go-version: 1.18.1
```

Comment:
can you also switch both of these to use go-version-file?
you'll have to move the checkout before setup-go